common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,156 @@
+"""
+Acceptance Tests for Course Information
+"""
+from common.test.acceptance.pages.studio.course_info import CourseUpdatesPage
+from common.test.acceptance.tests.studio.base_studio_test import StudioCourseTest
+from flaky import flaky
+
+from ...pages.studio.auto_auth import AutoAuthPage
+from ...pages.studio.index import DashboardPage
+
+
+@flaky(25, 25)
```

Comment:
This is so that the test will run 25 times on Jenkins
[] Remove this before Squashing and Merging